app/views/webhooks/signing_secret.njk

```diff
@@ -0,0 +1,71 @@
+{% extends "layout.njk" %}
+
+{% block pageTitle %}
+  Webhook signing secret {{ webhook.callback_url }} - {{currentService.name}} {{ humanReadableEnvironment }} - GOV.UK Pay
+{% endblock %}
+
+{% block side_navigation %}
+  {% include "includes/side-navigation.njk" %}
+{% endblock %}
+
+{% block mainContent %}
+<div class="govuk-grid-column-two-thirds">
+  {{ govukBackLink({
+    text: "Back to Webhook",
+    classes: "govuk-!-margin-top-0",
+    href: formatFutureStrategyAccountPathsFor(routes.futureAccountStrategy.webhooks.detail, currentGatewayAccount.type, currentGatewayAccount.service_id, currentGatewayAccount.external_id, webhook.external_id)
+  }) }}
+
+  <h1 class="govuk-heading-l page-title">Manage signing secret</h1>
+
+  <p class="govuk-body">
+    The contents of Webhooks are encrypted with a signing secret. Only your application should store this secret to unencrypt Webhooks content.
```

Comment:
This text isn't really accurate. You don't use the signing key to 'unencrypt' the content, you use it to verify the authenticity of the message. Can we at least put a TODO here and maybe we can pick up the content with @NathanD-GDS et al?

Reply:
👍  this is lifted and shifted from some very early days stuff. I'll add it to a list of comments being addressed in the next pass